### PR TITLE
executor: a naive support for load data fields enclosed by

### PR DESCRIPTION
**Pull request by tiancaiamao**
A very naive way to support fields enclosed by xxx:

change 
`split(input, terminated)`
to 
`split(input, enclosed+terminated+enclosed)`

@zimulala @shenli

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @zimulala

**Comment by zimulala** [comment]
PTAL @hanfei1991

File: executor/write.go [hunk 1]
```diff
@@ -467,10 +489,8 @@ func escape(str []byte) []byte {
 	for i := 0; i < len(str); i++ {
 		c := str[i]
 		if c == '\\' && i+1 < len(str) {
-			var ok bool
-			if c, ok = escapeChar(str[i+1]); ok {
-				i++
-			}
+			c, _ = escapeChar(str[i+1])
```
Comment (shenli) [inline executor/write.go]: Why ignore ok？ Any test case covers this?

File: executor/write.go [hunk 2]
```diff
@@ -467,10 +489,8 @@ func escape(str []byte) []byte {
 	for i := 0; i < len(str); i++ {
 		c := str[i]
 		if c == '\\' && i+1 < len(str) {
-			var ok bool
-			if c, ok = escapeChar(str[i+1]); ok {
-				i++
-			}
+			c = escapeChar(str[i+1])
```
Comment (shenli) [inline executor/write.go]: What will the result be if we meet "\c"?

Comment (tiancaiamao) [inline executor/write.go]: \c is escaped to c

**Comment by shenli** [comment]
LGTM

**Comment by zimulala** [comment]
Please update the `docs-cn`.

**Pull request merged**